docs/source/quantization/int8_kv_cache.rst

```diff
@@ -0,0 +1,52 @@
+.. _int8_kv_cache:
+
+INT8 KV Cache
+==================
+
+The kv cache is quantized to INT8 dtype from float/fp16/bflaot16 to save GPU memory.
+To use it, you first need to export scales and zero points with a calibration dataset like pileval and save these quantization parameters at a certain path.
+Then you can enable the int8 kv cache in the vllm settings.
+
+
+Here is an example of how to export quantization scales and zero points:
+
+First, you should capture kv cache states for subsequent calculation of scales and zero points.
+
+.. code-block:: console
+
+    $ python3 vllm/kv_quant/calibrate.py --model facebook/opt-125m --calib_dataset pileval 
```

Comment:
calibrate.py does not support opt model yet. I suggest you replace opt with llama model.

Reply:
fixed

Reply:
Advice: Add `path` args to function `get_pileval` to support load dataset from a given directory.

Reply:
'path' arg has been added to get_pileval